docs/using/handlers.md

```diff
@@ -0,0 +1,164 @@
+a topology to the potential keys (`TopologyKey` to `PotentialKey` mapping). The
+`.potentials` attribute maps the potential keys to the potentials
+(`PotentialKey` to `Potential`). This allows many topology keys to map to the
+same `Potential` by sharing a `PotentialKey`. If the `Potential` is updated,
+all the places in the topology where it is used are updated immediately.
+Despite this, getting the `Potential` for a place in the topology is a constant
+time operation.
+
+Each potential handler inherits from the base [`PotentialHandler`] class and
+describes a single type of parameter from a single source. Potential handlers
+for SMIRNOFF force fields are found in the [](openff.interchange.components.smirnoff)
+module, while those for Foyer are found in the [](openff.interchange.components.foyer).
+module.
+
+## Inspecting an assigned parameter
+
+Construct a simple `Interchange`
+
+```python
+>>> from openff.interchange import Interchange
+>>> from openff.toolkit import Molecule, ForceField
+>>>
+>>> ethane = Molecule.from_smiles("CC")
+>>> sage = ForceField("openff-2.0.0.offxml")
+>>> box = box=[[1,0,0], [0,1,0], [0,0,1]]
+>>> interchange = Interchange.from_smirnoff(sage, [ethane], box=box)
+
+```
+
+The [`Interchange.handlers`] attribute maps names to the corresponding handler:
+
+```python
+>>> interchange.handlers.keys()  # doctest: +NORMALIZE_WHITESPACE,+ELLIPSIS
+dict_keys(['Bonds', 'Constraints', 'Angles', 'ProperTorsions',
+    'ImproperTorsions', 'vdW', 'Electrostatics'])
+>>> # Ethane has no improper torsions, so both maps will be empty
+>>> interchange.handlers['ImproperTorsions']  # doctest: +NORMALIZE_WHITESPACE,+ELLIPSIS
+SMIRNOFFImproperTorsionHandler(type='ImproperTorsions',
+    expression='k*(1+cos(periodicity*theta-phase))',
+    slot_map={},
+    potentials={})
+
+```
+
+In the bond handler for example, each pair of bonded atoms maps to one of two
+potential keys, one for the carbon-carbon bond, and the other for the
+carbon-hydrogen bonds. It's clear from the SMIRKS codes that atoms 0 and 1 are
+the carbon atoms, and atoms 2 through 7 are the hydrogens:
+
+```python
+>>> interchange.handlers['Bonds'].slot_map  # doctest: +NORMALIZE_WHITESPACE,+ELLIPSIS
+{TopologyKey(atom_indices=(0, 1), ...): PotentialKey(id='[#6X4:1]-[#6X4:2]', ...),
+ TopologyKey(atom_indices=(0, 2), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...),
+ TopologyKey(atom_indices=(0, 3), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...),
+ TopologyKey(atom_indices=(0, 4), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...),
+ TopologyKey(atom_indices=(1, 5), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...),
+ TopologyKey(atom_indices=(1, 6), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...),
+ TopologyKey(atom_indices=(1, 7), ...): PotentialKey(id='[#6X4:1]-[#1:2]', ...)}
+
+```
+
+:::{hint}
+Which atom indices represent hydrogens bonded to carbon atom 0, and which are
+bonded to carbon atom 1?
+:::
```

Comment:
I like what this adds to the reading of the example, but something is a little strange in how it's presented. A box titled "Hint" got me thinking if the text had a question in it like in a textbook, but it doesn't. I'd suggest either 
* inserting a question: "From just this mapping, can you find the index of the hydrogen atom bonded to carbon atom 1?" or
* rephrasing it to be descriptive: "notice how with only this information, you can tell that i.e. the hydrogen atom with index 5 is bonded to the carbon with index 1"

Reply:
I've changed it to a question admonition (support for which was already hidden in the theme. and by "hidden" I mean "written by me and forgotten about")